resilience4j-spring-boot3/src/main/java/io/github/resilience4j/springboot3/timelimiter/autoconfigure/AbstractTimeLimiterConfigurationOnMissingBean.java

```diff
@@ -58,6 +58,12 @@ public CompositeCustomizer<TimeLimiterConfigCustomizer> compositeTimeLimiterCust
         return new CompositeCustomizer<>(customizers);
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public TimeLimiterProperties timeLimiterProperties() {
```

Comment:
Why do we need to add TimeLimiterProperties here?

Reply:
I added the `TimeLimiterProperties` bean because the `PlainObjectBulkheadAspectExt` depends on the `TimeLimiterRegistry`, the `TimeLimiterRegistry` itself requires `TimeLimiterConfigurationProperties` (which extends `TimeLimiterProperties`) for its configuration, and without this bean declaration, the dependancy chain breaks and causes failures in multiple tests, specifically in `BulkheadConfigurationOnMissingBeanTest`